.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,15 @@
 <!-- Briefly describe the change and why it was required. -->
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Reviewer Checklist
+- [ ] PR title and description are accurate and comprehensive. The description should include motivation for the change, design decisions and concerns.
+- [ ] PR should be focused. No unnecessary changes were introduced in this PR.
+- [ ] No breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes are introduced.
```

Comment:
```suggestion
- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
```

Reply:
An API breaking change would require a major version bump, so should this read ~ "A breaking API change is allowed only in a major version release"?